opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -113,6 +138,17 @@
          */
         SpanBuilder addReference(String referenceType, SpanContext referencedContext);
 
+        /**
+         * Remove any explicit (e.g., via {@link SpanBuilder#addReference(String,SpanContext)}) or implicit (e.g., via
+         * {@link ActiveSpanSource#active}) references to parent / predecessor SpanContexts, thus making the built
+         * Span a "root" of a Trace tree/graph.
+         *
+         * <p>
+         * Subsequent calls to {@link SpanBuilder#addReference(String, SpanContext)} /
+         * {@link SpanBuilder#asChildOf(Span)} / etc are permitted and behave as per usual.
+         */
+        SpanBuilder asRoot();
```

Comment:
+1. I would prefer something like "detached" or even more explicit "detachFromActiveSpan".

And I am `-1` on "Remove any _explicit_ references" - if I am adding them explicitly, I mean it, and it shouldn't matter if I call addReference() before or after detachFromActiveSpan(). Incidentally, calling addReference() makes detachFromActiveSpan() redundant according to the current API.